lib/data_hygiene/policy_tagger.rb

```diff
@@ -0,0 +1,80 @@
+class PolicyTagger
+  def self.process_from_csv(csv_location, logger: Logger.new(nil))
+    CSV.foreach(csv_location, headers: true) do |row|
+      policies_to_remove = row.fetch("policies_to_remove") ? policies_to_remove.split(" ") : []
```

Comment:
`fetch` doesn't work how you're expecting here. You can reduce these down to `policies_to_remove = row.fetch("policies_to_remove").split`. `fetch` will guard against it being missing and raise, and `split` can be used without arguments. It defaults to splitting on spaces.

Alternatively you should guard against the values of `policies_to_remove` and `policies_to_add` in the initialiser *if* an instance of `PolicyTagger` is meaningless without values for those being provided.